ldm/invoke/patchmatch.py

```diff
@@ -0,0 +1,44 @@
+'''
+This module defines a singleton object, "patchmatch" that
+wraps the actual patchmatch object. It respects the global
+"try_patchmatch" attribute, so that patchmatch loading can
+be suppressed or deferred
+'''
+from ldm.invoke.globals import Globals
+import numpy as  np
+
+class Patchmatch:
```

Comment:
Shouldn't this be `PatchMatch`?

(I'm very triggered that the PEP 8 style guidelines [refer to](https://peps.python.org/pep-0008/#descriptive-naming-styles) "CamelCase" where "Pascal Case" is the more appropriate/correct name.)

Reply:
Fixed, and tested.